metrics/storage/blogbench.sh

```diff
@@ -39,7 +39,7 @@ function main() {
 
 	metrics_json_init
 
-	local output=$(docker run --rm --runtime=$RUNTIME $IMAGE $CMD)
+	local output=$(docker run --cpus 4 --rm --runtime=$RUNTIME $IMAGE bash -c ''"$CMD"'')
```

Comment:
@devimc any particular reason why we are using 4 cpus?

Reply:
containers with virtiofs get stuck if there is only 1 cpu, maybe we should file an issue :sweat_smile:

Reply:
@devimc yes an issue and maybe at the top just put a comment about that and reference the issue

Reply:
@devimc just please add that and for me it looks the rest fine thanks

Reply:
@GabyCT done, thanks